### ISSUE TITLE
checkconstraints does not work for emulated architectures

### ISSUE THREAD

**Issue by andreas-schwab** [openSUSE/open-build-service#14846]
$ osc rblt . kernel-default RISCV riscv64
package build was not possible:

no compliant workers (constraints mismatch hint: hardware:disk)

Please adapt your constraints.
$ osc checkconstraints . kernel-default RISCV riscv64 | wc -l
1205

**Comment by krauselukas** [comment]
@andreas-schwab This issue was discussed on the mailing list right? Can I close it?

**Comment by adrianschroeter** [comment]
no, this is still an open issue (but not a webui issue, so please do not touch it).

**Comment by krauselukas** [comment]
@adrianschroeter alright, adding the backend label
